src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double secondLeg, double firstLeg) {
```

Comment:
The constructor parameters `secondLeg` and `firstLeg` are in the reverse order of the field declarations. It's a good practice to maintain the same order for clarity and to avoid confusion.